parse_name_from_email.gemspec

```diff
@@ -19,7 +19,13 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'activesupport'
+  if RUBY_VERSION >= '2.3'
+    spec.add_dependency 'activesupport'
+  elsif RUBY_VERSION >= '2.1'
+    spec.add_dependency 'activesupport', '~> 4.1'
+  elsif RUBY_VERSION < '2.1'
+    spec.add_dependency 'activesupport', '~> 3.2'
+  end
```

Comment:
Hi, just create a `Gemfile` for each ruby version, like [this](https://github.com/sshaw/itunes_store_transporter_web/tree/8af829618b45c0c03e2ea1c571ee2569061b153e/gemfiles). Then update your `.travisci.yml` file, like [this](https://github.com/sshaw/itunes_store_transporter_web/blob/8af829618b45c0c03e2ea1c571ee2569061b153e/.travis.yml).